maintainer-policies-and-guidelines.md

```diff
@@ -0,0 +1,23 @@
+# Maintainer Policies and Guidelines
+
+This document aims to clarify hard policies that Maintainers have to follow as well as less-enforced guidelines that are recommended to be followed. The goal is to help Maintainers understand what is expected of them and what actions should be done to reduce the chances of conflict within the Maintainer team.
+
+## Policies
+
+- `c` tags in 1.21+ NeoForge should be kept in sync with Fabric's `c` tags as close as possible. New `c` tags or changes to existing `c` tag should by ran by Fabric to ensure they will agree to make the same changes. `c` tags are to be a standard between loaders with Fabric being our main partner in this effort.
+
+  - Failure to meet this expectation will result in the change being undone so NeoForge matches Fabric again and the incident reviewed to understand how we failed to communicate with Fabric.
```

Comment:
This reads very punitive, and adds little value. Yes mistakes do happen and we will try to correct them. An immediate revert may or may not be the solution, especially if the mistake is only noticed later. To be determined on a case-by-case basis, IMO.